### PR TITLE
Add textmode release notes

### PR DESCRIPTION
**Pull request by dzedro**
gnome
http://10.100.98.90/tests/1091/modules/releasenotes/steps/1
textmode
http://10.100.98.90/tests/1090/modules/releasenotes/steps/1

### CODE REVIEW

File: tests/installation/releasenotes.pm [hunk 1]
```diff
@@ -5,8 +5,15 @@ use testapi;
...
-    assert_and_click 'release-notes-tab-sle';   # click on first SLES tab
+    if (check_var('VIDEOMODE', 'text')) {
+        send_key "alt-l";   # open release notes window
+        send_key 'alt-s';   # jump to first tab
+        assert_screen 'release-notes-tab-sle';
+    }
+    else {
+        assert_and_click 'release-notes-button';    # open release notes window
+        assert_and_click 'release-notes-tab-sle';   # click on first SLES tab
+    }
```
Comment (lnussel) [inline tests/installation/releasenotes.pm]: do we really need both? in the ui the shortcuts should also work, right?

Comment (dzedro) [inline tests/installation/releasenotes.pm]: Unfortunately in graphical installation there is no shortcut for release notes.

Comment (nilxam) [inline tests/installation/releasenotes.pm]: hmm...yes, IIRC "Help" and "Release Note" button doesn't provide the shortcut in graphical mode since SLE12-GA

Comment (lnussel) [inline tests/installation/releasenotes.pm]: Lets's see if that is a bug or feature:
https://bugzilla.suse.com/show_bug.cgi?id=936743

**Comment by coolo** [comment]
shortcuts are expensive - so it's fine that buttons not being on the critical path don't have one. Just IMO

**Pull request merged**